src/sass/chrome.scss

```diff
@@ -69,6 +70,33 @@ aside {
     }
 }
 
+// Beta Banner
```

Comment:
Can we just make this a different .scss file and import it into the beta banner component?